### PR TITLE
Python: add create/list/delete for node and edge

### PR DESCRIPTION
**Pull request by adrianlzt**
Following the current implementation for noderule/edgerule in python and
node/edge create in the "skydive client".

Data validation is done by the server.

skip go-fmt
skip unit-tests
skip compile-tests
skip functional-tests-backend-elasticsearch
skip functional-tests-backend-orientdb
skip functional-hw-tests
skip scale-tests
skip cdd-overview-tests
skip k8s-tests

### CODE REVIEW

**Comment by adrianlzt** [comment]
Maybe this client should return a different Exception if the problem is a conflicting id?

**Comment by lebauce** [comment]
run python-tests

**Comment by lebauce** [comment]
run python-tests

**Comment by adrianlzt** [comment]
Tests are failing because an error in one of the steps of the Dockerfiile.
Looks like and old repo server

File: contrib/python/api/skydive/rest/client.py [hunk 1]
```diff
@@ -227,6 +231,65 @@ def edgerule_delete(self, rule_id):
         path = "/api/edgerule/%s" % rule_id
         self.request(path, method="DELETE")
 
+    def node_create(self, node_id=None, host=None, metadata=None):
+        now = int(time())
+        if not node_id:
+            edge_id = str(uuid4())
```
Comment (masco) [inline contrib/python/api/skydive/rest/client.py:237]: node_id

**Comment by lebauce** [comment]
@adrianlzt Some pycodingstyle and flake8 tests are failing. Feel free to cherry pick this https://github.com/lebauce/skydive/commit/a605b29a047556ecb0a12ee0a2e63efeb42e3f96

**Comment by adrianlzt** [comment]
New pypi release? :D
https://pypi.org/project/skydive-client/#history

**Comment by safchain** [comment]
done :) https://pypi.org/project/skydive-client/0.9.0/

**Pull request merged**